gufe/tests/test_chemicalstate.py

```diff
@@ -0,0 +1,51 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import pytest
+from openff.toolkit.topology import Molecule
+
+import gufe
+
+
+@pytest.fixture
+def prot_comp(PDB_181L_path):
+    yield gufe.ProteinComponent.from_pdbfile(PDB_181L_path)
```

Comment:
Can you replace this with a fully prepared protein with hydrogens please? Ping me if you want me to generate it, should only take a couple of minutes (I usually use the protein plus webserver + pymol for fixing any issues)

Reply:
Yeah please, ideally it'll be pretty close to what we actually use

Reply:
Cool I'll ping it across in a few, just need to check the coordinates match with the benzene modifications.

Reply:
@richardjgowers - I've added a new PDB and CIF files (181L-prepped.*) that contains 181L with protons, waters within 10 A of the positions where the benzene ligand would be, ACE and NME caps, and two Cl- ions which were part of the crystallographic artifacts.

Couple of things:

1. The ions aren't structural but were kept so that we had a direct test case for the presence of ions in the "protein" component.

2. The cap isn't necessary on one of the termini, but also good to have so we know it won't be rejected in real life.

Long term we should also add in a multi-chain test case too.

Reply:
@IAlibay can you just overwrite the old pdb&cif?

Reply:
sure

Reply:
Alright took some CI fixing, but done.